### DISCUSSION TITLE
Homeassistant Dashboard unresponsive with Browser extension installed

### DISCUSSION THREAD

**Discussion by Node815** [dani-garcia/vaultwarden#5178] [Q&A]
This was addressed here by the Bitwarden team, but since I use Vaultwarden, I wanted to see if it was migrated over by chance, I have the same symptoms:
https://github.com/bitwarden/clients/issues/9617

I am using the latest of Firefox and also Zen Browser and when I added the Bitwarden extension it causes the non responsive views in Home Assistant.   

To summarize, this happens when I leave the tab open for roughly 3 or 4 minutes to idle, then it starts happening.  Has anyone else encountered this?   I wanted to check before I wrote up an issue. :)

**Comment by BlackDex** [comment]
Vaultwarden doesn't do anything with the clients other then providing it data. The web-vault has nothing to do with any other client, so that is also no link. 

So it must be the browser extension it self then, or maybe an other extension which causes this. But i doubt it has anything to do Vaultwarden directly. 

Try disabling all extensions and turn them on one by one, starting with the Bitwarden extension and see what happens.

**Reply by Node815** [reply]
Fair enough, I did try all of that before asking here, and pin pointed it to the browser extension.  I know I'm not the only person having this issue, so I will see what else I can do.  I just wasn't sure if it was somehow tied to the web-vault which you confirm it wouldn't be .
